delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/acls/AclsImpl.scala

```diff
@@ -31,11 +32,19 @@ final class AclsImpl private (
     index: AclsCache
 ) extends Acls {
 
+  private val minimum: Set[Permission] = permissions.minimum
+
   override def fetch(address: AclAddress): UIO[Option[AclResource]] =
-    agg.state(address.string).map(_.toResource).named("fetchAcl", moduleType)
+    agg.state(address.string).map(_.toResource(address, minimum)).named("fetchAcl", moduleType)
 
   override def fetchWithAncestors(address: AclAddress): UIO[AclCollection] =
```

Comment:
It looks identical to the method in the dummy, maybe we could put it directly in the Acls ?

Reply:
There is few duplication, but in this case is minimal. I don't think it really helps for readability to put it in Acls.